src/packages/core/repository/detail/detail-repository-base.ts

```diff
@@ -87,8 +87,8 @@ export abstract class UmbDetailRepositoryBase<
 			this.#detailStore?.append(createdData);
 
 			// TODO: how do we handle generic notifications? Is this the correct place to do it?
-			const notification = { data: { message: `Created` } };
-			this.#notificationContext!.peek('positive', notification);
+			// const notification = { data: { message: `Created` } };
+			// this.#notificationContext!.peek('positive', notification);
```

Comment:
I would like some sparing on what HQ finds would be the best way to handle this. I've disabled the notification here because when uploading many files the right hand side of the screen just becomes a green wall of `Created` messages. I think the best way for the drag and drop feature is to not have any `Created` messages at all and only the progress and failure notification messages show up but is it feasible to move the notification from here closer to the places we would like to show it I can see this method has many references.